mozci/util/taskcluster.py

```diff
@@ -73,3 +75,26 @@ def get_index_url(index_path):
 def find_task_id(index_path, use_proxy=False):
     response = _do_request(get_index_url(index_path))
     return response.json()["taskId"]
+
+
+def get_task_url(task_id):
+    return liburls.api(
+        PRODUCTION_TASKCLUSTER_ROOT_URL, "queue", "v1", f"task/{task_id}"
+    )
+
+
+def get_task(task_id, use_proxy=False):
+    response = _do_request(get_task_url(task_id))
+    return response.json()
+
+
+def get_tasks_in_group(group_id):
+    tasks = []
+
+    def _save_tasks(response):
+        tasks.extend(response["tasks"])
+
+    queue = taskcluster_config.get_service("queue")
+    queue.listTaskGroup(group_id, paginationHandler=_save_tasks)
+
+    return tasks
```

Comment:
I could have implemented this using requests, but this is way easier.

Reply:
It's not that hard :p 
https://searchfox.org/mozilla-central/source/taskcluster/taskgraph/util/taskcluster.py#327

But agree that using the API seems nicer. Should we convert everything to use the API?

Reply:
> It's not that hard :p

Yeah, but it is cleaner (i.e. less code) this way :P

> But agree that using the API seems nicer. Should we convert everything to use the API?

Yeah, I'll file it as a good first issue.

Reply:
I filed https://github.com/mozilla/mozci/issues/306.